deploy/devnet/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   archive_node:
-    image: sportchain-node
+    image: docker.io/atletanetwork/atleta-node:devnet-latest
```

Comment:
Are you sure `devnet-latest` will work? Isn't it `latest` or a particular tag?

Reply:
Also, do we need `docker.io/` here?

Reply:
Devnet-latest must work as it is the image we are tagging and pushing along with versioned image
Docker.io is not necessary, removed